docs/configuration.md

```diff
@@ -456,6 +456,33 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
+  <td><code>spark.sql.repl.eagerEval.enabled</code></td>
+  <td>false</td>
+  <td>
+    Enable eager evaluation or not. If true and REPL you are using supports eager evaluation,
+    dataframe will be ran automatically. HTML table will feedback the queries user have defined if
```

Comment:
`dataframe` -> `DataFrame/Dataset`

What is `HTML table`? Is the term used in Jupyter only?

Reply:
The HTML table is generated by `_repr_html_`, isn't Jupyter only term. `_repr_html` is the rich display support for IPython in notebook and Qt console. I think it can be used in other place but currently I just test this in Jupyter. I re-write the doc, please check is it appropriate, thanks.